lib/fs.js

```diff
@@ -476,7 +476,9 @@ function open(path, flags, mode, callback) {
   } else {
     mode = parseFileMode(mode, 'mode', 0o666);
   }
+
```

Comment:
Non-blocking nit: Might want to remove this unrelated whitespace change and the one two lines later to prevent them from being the source of a merge conflict when this is cherry-picked.